infrastructure/src/main/java/org/corfudb/infrastructure/ManagementAgent.java

```diff
@@ -390,7 +403,7 @@ private SequencerMetrics queryLocalSequencerMetrics(Layout layout) {
         SequencerMetrics sequencerMetrics;
         // This is an optimization. If this node is not the primary sequencer for the current
         // layout, there is no reason to request metrics from this sequencer.
-        if (layout.getSequencers().get(0).equals(getLocalEndpoint())) {
+        if (layout.getSequencersNodes().get(0).equals(getLocalEndpoint())) {
```

Comment:
should be either getSequencers or getSequencerNodes

Reply:
`getSequencers()` marked as deprecated, will be deleted in future versions